### ISSUE TITLE
Cannot run composer install

### ISSUE THREAD

**Issue by roelywoely** [anttiviljami/wordpress-heroku-docker-project#8]
I got stuck on the composer install command:

```
root@a68f92021c70:/app/user# composer install
Do not run Composer as root/super user! See https://getcomposer.org/root for details
> test -f /app/.heroku/mysql-client-core.deb || wget http://security.ubuntu.com/ubuntu/pool/main/m/mysql-5.7/mysql-client-core-5.7_5.7.20-0ubuntu0.16.04.1_amd64.deb -qO /app/.heroku/mysql-client-core.deb
> test -f /app/.heroku/mysql-core.deb || wget http://security.ubuntu.com/ubuntu/pool/main/m/mysql-5.7/mysql-client-5.7_5.7.20-0ubuntu0.16.04.1_amd64.deb -qO /app/.heroku/mysql-client.deb
Script test -f /app/.heroku/mysql-core.deb || wget http://security.ubuntu.com/ubuntu/pool/main/m/mysql-5.7/mysql-client-5.7_5.7.20-0ubuntu0.16.04.1_amd64.deb -qO /app/.heroku/mysql-client.deb handling the pre-install-cmd event returned with error code 8
root@a68f92021c70:/app/user#
```

Apparently the security.ubuntu.com links are dead.

**Comment by joshghent** [comment]
@roelywoely Fixed in https://github.com/anttiviljami/wordpress-heroku-docker-project/pull/9

Encourage @anttiviljami to merge it in once the TravisCI passes.

**Comment by anttiviljami** [comment]
Thanks. Fixed in #9
